src/componentLibrary/theme/theme.ts

```diff
@@ -60,6 +60,10 @@ const theme = {
     secondary: {
       bg: 'var(--colors-cta-secondary)'
     }
+  },
+  modal: {
+    bg: 'var(--colors-modal-bg)',
+    transitionDuration: 100
```

Comment:
Would it make more sense to have transitions set as their own thing? I think we're likely to want consistent transitions, rather than transitions connected specifically to components.